scalafmt-core/shared/src/main/scala/org/scalafmt/config/ScalafmtConfig.scala

```diff
@@ -140,19 +140,21 @@ case class ScalafmtConfig(
     fileOverride: Conf.Obj = Conf.Obj.empty,
     xmlLiterals: XmlLiterals = XmlLiterals()
 ) {
-  lazy val alignMap: Map[String, Seq[AlignToken.Matcher]] =
+  private[scalafmt] lazy val alignMap: Map[String, Seq[AlignToken.Matcher]] =
```

Comment:
Should we really interact with this class at all? It's not part of the API within scalafmt-dynamic.

Reply:
Looks like this was originally the plan: https://github.com/JetBrains/intellij-scala/blob/idea212.x/scala/scala-impl/src/org/jetbrains/plugins/scala/lang/formatting/scalafmt/dynamic/ScalafmtDynamic.scala#L21

Any idea what happened there?

Reply:
`private` doesn't help much if they use reflection.

Reply:
> Should we really interact with this class at all? It's not part of the API within scalafmt-dynamic.

intellij seems to have copied some version of ScalafmtReflectConfig, and that class does call some internal methods.

@unkarjedy is there any other solution which uses externally an published interface? should we add your indents into the class so you can access them?

Reply:
> `private` doesn't help much if they use reflection.

didn't know that... but at least it shows intent :)